tutoroars/templates/oars/jobs/init/oars_init_schemas_tables_users.sh

```diff
@@ -1,13 +1,3 @@
-echo "Install clickhouse-client..."
-apt-get install -y apt-transport-https ca-certificates dirmngr
-apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754
-
-echo "deb https://packages.clickhouse.com/deb stable main" | tee \
-    /etc/apt/sources.list.d/clickhouse.list
-apt-get update
-
-apt-get install -y clickhouse-client
-
```

Comment:
As this now runs as a clickhouse job, we do not need to install any of those packages. It speed ups the initialization tasks.

Reply:
Awesome! 🎉